examples/step-55/doc/intro.dox

```diff
@@ -0,0 +1,71 @@
+<a name="Intro"></a>
+<h1>Introduction</h1>
+
+<h3> Stokes Problem </h3>
+
+The purpose of this tutorial is to create an efficient linear solver for the Stokes equation and compare it to alternative approaches. Using FGMRES with geometric multigrid as a precondtioner for the velocity block, we see that the linear solvers used in Step-22 cannot keep up since multigrid is the only way to get $O(n)$ solve time. Using the Timer class, we collect some statistics to compare setup times, solve times, and number of iterations. We also compute errors to make sure that what we have implemented is correct.
```

Comment:
github unfortunately doesn't break long lines when showing patches. This makes it difficult to review this file. Can you break long lines at 80 characters? In emacs, you can do this by hitting `ESC-q` on long lines.